content/docs/user-guide/experiment-management/persisting-experiments.md

```diff
@@ -0,0 +1,87 @@
+# Persisting Experiments
+
+DVC runs experiments outside of the standard Git stage/commit workflow for quick
+iteration. In this section we'll see how to bring the experiments we run to the
+standard Git workflow with `dvc exp branch` and `dvc exp apply`.
+
+## Create a Git branch for your experiment
+
+You can use `dvc exp branch` to create a new branch from the experiment and keep
+all code and artifacts separate from your current one.
```

Comment:
Yeah I'd definitely keep the `apply` section first and `branch` after.

Reply:
@dberenbaum so you recommend `exp branch` be introduced first as we want to encourage it more? Thanks